support-frontend/assets/pages/supporter-plus-landing/supporterPlusRouter.tsx

```diff
@@ -59,6 +65,22 @@ const router = () => {
 		<SupporterPlusInitialLandingPage thankYouRoute={thankYouRoute} />
 	);
 
+	const isThreeTierAndOneOffUrlParam = () => {
```

Comment:
Could this just be a value rather than method?

Reply:
yep 👍